backend/infrahub/graphql/mutations/ipam.py

```diff
@@ -48,7 +48,7 @@ async def validate_namespace(
         namespace_id = namespace.id
     elif "hfid" in data["ip_namespace"]:
         namespace = await registry.manager.get_one_by_hfid(
-            db=db, branch=branch, kind=InfrahubKind.IPNAMESPACE, hfid=data["ip_namespace"]["hfid"]
+            db=db, branch=branch, kind=InfrahubKind.IPNAMESPACE, hfid=data["ip_namespace"]["hfid"], raise_on_error=True
```

Comment:
This change is required to make the type checkers happy, it looks like we always assumed that we'd get a return value here as we on line 53 tro to assign namespace_id from `namespace.id` if no object was found and we'd returned a `None` value we'd instead get an `AttributeError`. Before when we returned an `Any` the typechecker was not able to spot this. As such it should be fine to always raise an error if we don't find the node here.